### PR TITLE
Get emails from clickhouse v1

### PR DESCRIPTION
**Pull request by ynang**

### CODE REVIEW

File: dags/oss_know/libs/email_address/init_email_address_from_ck.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    ck = CKServer(host=clickhouse_server_info["HOST"],
+                  port=clickhouse_server_info["PORT"],
+                  user=clickhouse_server_info["USER"],
+                  password=clickhouse_server_info["PASSWD"],
+                  database=clickhouse_server_info["DATABASE"])
+
+    gits_sql = "SELECT DISTINCT author_email,committer_email FROM gits"
+    gits_email_from_ck = ck.execute_no_params(gits_sql)
+    all_email_address_dict = {}
+    for author_email, committer_email in gits_email_from_ck:
```
Comment (crystaldust) [inline dags/oss_know/libs/email_address/init_email_address_from_ck.py]: A possible better solution: get two types of emails with DISTINCT from the table. Make them 2 sets, then get the union

File: dags/oss_know/libs/email_address/init_email_address_from_ck.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        for item in author_email, committer_email:
+            if item:
+                all_email_address_dict[item] = 0
+    if all_email_address_dict:
```
Comment (crystaldust) [inline dags/oss_know/libs/email_address/init_email_address_from_ck.py]: Unnecessary check

File: dags/oss_know/libs/email_address/init_email_address_from_ck.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    github_profile_email_from_ck = ck.execute_no_params(github_profile_sql)
+    for email, id in github_profile_email_from_ck:
+        if email:
+            all_email_address_dict[email] = id
+
+    # count = 0
+    insert_email_address_data_params = []
+    if all_email_address_dict:
+        for k, v in all_email_address_dict.items():
+            insert_email_address_data = []
```
Comment (crystaldust) [inline dags/oss_know/libs/email_address/init_email_address_from_ck.py]: Check if it's possible to insert a key-value pair

Comment (ynang) [reply dags/oss_know/libs/email_address/init_email_address_from_ck.py]: Done,Thanks:)

File: dags/oss_know/libs/email_address/init_email_address_from_ck.py (hunk 2 above)
Comment (ynang) [reply dags/oss_know/libs/email_address/init_email_address_from_ck.py]: Done,Thanks:)

**Pull request merged**